content/post/051-new-ipfs-release-process.md

```diff
@@ -0,0 +1,209 @@
+---
+date: 2019-07-18
+url: 2019-07-18-ipfs-release-process
+tags: IPFS, go-ipfs, release process, 
+title: Improving the IPFS Release Process
+author: Steven Allen, Alan Shaw, David Dias, Molly Mackinlay
+---
+
+go-ipfs is introducing a new release cycle and process to ensure reliable releases.
+
+## Background
+
+The past three releases have had a few critical regressions, several of which
+would have been caught by better release testing and released faster with an
```

Comment:
```suggestion
would have been caught by better release testing, or released faster fixes with an
```

Reply:
Hm. That's still not _quite_ correct. I've changed it to:

> several of which would have been caught by better release testing and have had fixes released faster with an improved release process.

But that's a bit of a mouthful.

Reply:
How about:

> The past three releases have introduced a few critical regressions (since
fixed), several of which would have been caught by better release. Baring that,
a better release process would have enabled us to quickly release fixes for
these regressions.

Reply:
Actually, I've just reworked this section.